### PR TITLE
Mention SubViewport.render_target_clear_mode property is intended for 2D usage

### LINKED ISSUE #33831: Viewport "Clear mode" isn't disabling clearing the render target
**Godot version:**
v3.2 - 28613ab8c

**OS/device including version:**
Ubuntu Linux 19.10
Linux Kernel 5.3.0.19-generic

**Issue description:**

When setting a Viewport render target clear mode to never, after clearing it for the first time, it keeps cleaning the buffer on each frame, not accumulating renders.

**Steps to reproduce:**

1. Create a viewport
2. Change Clear mode to Next frame
3. Disable Next frame (as it isn't deactivated automatically)
4. Framebuffer on the render target keeps getting cleared each frame

**Minimal reproduction project:**
N/A

### PR DESCRIPTION
**Pull request by Yetizone**
Fixes https://github.com/godotengine/godot/issues/33831

### CODE REVIEW

File: doc/classes/SubViewport.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	</methods>
 	<members>
 		<member name="render_target_clear_mode" type="int" setter="set_clear_mode" getter="get_clear_mode" enum="SubViewport.ClearMode" default="0">
-			The clear mode when the sub-viewport is used as a render target.
+			The clear mode when the sub-viewport is used as a render target. Note this property is intended for 2D usage.
```
Comment (NathanLovato) [inline doc/classes/SubViewport.xml:13]:
```suggestion
			The clear mode when viewport used as a render target.
			[b]Note:[/b] this property is intended for 2D usage.
```

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Suggesting a formatting change for consistency with the rest of the class ref

File: doc/classes/SubViewport.xml (hunk 1 above)
Comment (Yetizone) [reply doc/classes/SubViewport.xml:13]: I've noticed that in your comment here you're also suggesting to change the first sentence, i.e. replace "the sub-viewport" with "viewport" and remove "is". 
Just wanted to double check if you're sure about the changes or if they were perhaps unintentional :sweat_smile:

Comment (NathanLovato) [reply doc/classes/SubViewport.xml:13]: It was unintentional, yes. I'm fixing it

**Review by NathanLovato — CHANGES_REQUESTED** [review]
You added some commits to your PR inadvertently

**Comment by akien-mga** [comment]
Needs a rebase as I exceptionally force-pushed a rebase of the `master` branch to fix the unwanted "wip1" commit shown above.

Then the same style fixes as in #41713 can be done too.

**Comment by NathanLovato** [comment]
I edited the suggested change to reflect the style guide

**Comment by Yetizone** [comment]
Okay, rebased and adjusted style. 

Thanks for letting me know what happened, I was a bit confused when I saw the PR suddenly had 1900+ additions :sweat_smile:

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**